src/expression_utils.jl

```diff
@@ -19,36 +22,9 @@ function is_escaped_expr(expr)
     return (expr isa Expr) && (expr.head == :escape) && (length(expr.args) == 1)
 end
 
-### Parameters/Species/Variables Symbols Correctness Checking ###
-
-# Throws an error when a forbidden symbol is used.
-function forbidden_symbol_check(v)
-    !isempty(intersect(forbidden_symbols_error, v)) &&
-        error("The following symbol(s) are used as species or parameters: " *
-              ((map(s -> "'" * string(s) * "', ",
-                  intersect(forbidden_symbols_error, v))...)) *
-              "this is not permitted.")
-    nothing
-end
```

Comment:
This is not needed, because the check is already carried out in `Reaction` and `ReactionSystem`.